api/api.go

```diff
@@ -672,7 +677,11 @@ mkdir -p "/etc/sv/$APP_NAME/env"
 # Create the run script for the app
 cat << EOF > /etc/sv/$APP_NAME/run
 #!/bin/bash
-exec 2>&1 chpst -e /etc/sv/$APP_NAME/env $DOCKER run --rm --name $APP_NAME-run -p 9090:$PORT "$IMAGE"
+exec 2>&1 $DOCKER run \
+{{range .Variables -}}
+	-env {{.Name}}="{{.Value}}" \
```

Comment:
`--env` not `-env`

If you leave off the `="{{.Value}}"`, [Docker will pick up the env var from the environment](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e-env-env-file), which if we invoke it with `chpst -e /etc/sv/$APP_NAME/env` and add the [env vars to that dir per](http://smarden.org/runit/chpst.8.html) will populate the environment the way we want. So we'll need to do that. I would prefer to do it the "runit" way by adding the env vars to the `chpst`-compatible dir than bake them into the `run` script.